src/gsp.py

```diff
@@ -80,11 +88,17 @@ def get_all_available_forecasts(
     - **end_datetime_utc**: optional end datetime for the query. e.g '2023-08-12 14:00:00+00:00'
     """
 
-    logger.info(f"Get forecasts for all gsps. The option is {historic=} for user {user}")
-
-    if gsp_ids is not None:
+    if isinstance(gsp_ids, str):
         gsp_ids = [int(gsp_id) for gsp_id in gsp_ids.split(",")]
 
+    if is_fake:
+        if gsp_ids is None:
+            gsp_ids = [int(gsp_id) for gsp_id in range(GSP_TOTAL)]
```

Comment:
I think this should be `range(1, GSP_TOTAL)` and same with below. Its becasue (weirdly) `gsp_id=0` is the national forecast